core/services/vrf/helpers_test.go

```diff
@@ -1,7 +1,32 @@
 package vrf
 
-import "math/big"
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
 
 func GenerateProofWithNonce(secretKey, seed, nonce *big.Int) (*Proof, error) {
 	return generateProofWithNonce(secretKey, seed, nonce)
 }
+
+func GenerateProofResponseWithNonce(secretKey *big.Int, preSeed *big.Int,
```

Comment:
This wrapping makes my head hurt.

Reply:
Stylistic policies like line-length wrapping or the complexity threshold for including [a function-level comment](https://duckduckgo.com/html?q=antirez%20%22function%20comments%22) are things I've thought about and have habits and opinions around, but I don't place much weight on them, so if you're interested in working on the VRF code, we can probably accommodate all of the concerns you've raised, but if you don't mind I'd rather not have them hold up this PR.